tasks/hooks/useAuth.ts

```diff
@@ -70,18 +70,20 @@ export const useAuth = () => {
   }
 
   useEffect(() => {
-    const idToken = Cookies.get(COOKIE_ID_TOKEN_KEY)
+    try {
+      const idToken = Cookies.get(COOKIE_ID_TOKEN_KEY)
 
-    const idTokenValid = idToken !== undefined && !isJwtExpired(idToken)
+      const idTokenValid = idToken !== undefined && !isJwtExpired(idToken)
 
-    if (!idTokenValid) Cookies.remove(COOKIE_ID_TOKEN_KEY)
+      if (!idTokenValid) Cookies.remove(COOKIE_ID_TOKEN_KEY)
 
-    if (idTokenValid) {
-      const user = tokenToUser(idToken)
-      if (!user) throw new Error('Cannot parse idToken to user')
-      setUser(user)
-      return
-    }
+      if (idTokenValid) {
+        const user = tokenToUser(idToken)
+        if (!user) throw new Error('Cannot parse idToken to user')
+        setUser(user)
+        return
+      }
+    } catch (err) {}
```

Comment:
```suggestion
    } catch (_) {}
```

Should we handle this? Do we have to ensure that some storages are cleared properly?

Reply:
Totally! We should not ignore this error 😄